MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -12221,23 +12221,24 @@ function start_a_blank_CASE_NOTE()
 	call navigate_to_MAXIS_screen("CASE", "NOTE")
 	DO
 		PF9
-		EMReadScreen case_note_check, 17, 2, 33
-		EMReadScreen mode_check, 1, 20, 09
-		If case_note_check <> "Case Notes (NOTE)" or mode_check <> "A" then
-            'msgbox "The script can't open a case note. Reasons may include:" & vbnewline & vbnewline & "* You may be in inquiry" & vbnewline & "* You may not have authorization to case note this case (e.g.: out-of-county case)" & vbnewline & vbnewline & "Check MAXIS and/or navigate to CASE/NOTE, and try again. You can press the STOP SCRIPT button on the power pad to stop the script."
-            BeginDialog Inquiry_Dialog, 0, 0, 241, 115, "CASE NOTE Cannot be Started"
-              ButtonGroup ButtonPressed
-                OkButton 185, 95, 50, 15
-              Text 10, 10, 190, 10, "The script can't open a case note. Reasons may include:"
-              Text 20, 25, 80, 10, "* You may be in inquiry"
-              Text 20, 40, 185, 20, "* You may not have authorization to case note this case (e.g.: out-of-county case)"
-              Text 5, 70, 225, 20, "Check MAXIS and/or navigate to CASE/NOTE, and try again. You can press the STOP SCRIPT button on the power pad to stop the script."
-            EndDialog
+		EMReadScreen case_note_open_check, 8, 1, 72
```

Comment:
Could we add a read of the mode and the message at the bottom of the screen? Maybe also the PW? Then if it fails and the error report is selected, we can add it to the 'script_run_lowdown' variable so we get additional information in any error report.

Reply:
Good idea. Added script_run_lowdown variable